workers/interfaces/WorkerUtils/navigator/008.worker.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+importScripts("/resources/testharness.js");
+test(function () {
+  let log = [];
+  for (const x in navigator) {
+    // skip functions, as they are settable
+    if (typeof navigator[x] === "function") continue;
+
+    // Trying to override a property should throw in strict mode
+    try {
+      navigator[x] = "";
+      log.push(x);
+    } catch (err) {}
```

Comment:
This needs to become an assert_throws_js or some such, no?

Reply:
yeah, that's better.